requirements/requirements-no-engine.yml

```diff
@@ -30,9 +30,9 @@ dependencies:
   - cloudpickle
   - boto3
   - pip:
-      # TODO(https://github.com/modin-project/modin/issues/5171): uncap xgboost
-      # when we can run tests on 1.7.0 without getting stuck.
-      - xgboost>=1.6.0,<1.7.0
+      # TODO(https://github.com/modin-project/modin/issues/5194): Uncap xgboost
+      # when we use collective instead of rabit.
+      - xgboost>=1.7.1,<2.0.0
```

Comment:
@vnlitvinov should we add the same pins for xgboost in `env_hdk.yml`?

Reply:
I have no idea as I don't have anything to do with HDK anymore, but I would like to have our requirements be as consistent as possible.

Reply:
I added the same requirement for hdk